src/main/resources/db/migration/V1_13__Create_case_data_tables.sql

```diff
@@ -21,39 +22,35 @@ create table if not exists darts.moj_case
   c_prosecutor       character varying array,
   c_defence          character varying array,
   retain_until_ts    timestamp with time zone,
-  r_version_label    varchar(32)
+  r_version_label    varchar(32),
+  i_superseded              BOOLEAN,
+  i_version                 SMALLINT);
 );
```

Comment:
The LLD said we don't need i_superseded' and 'r_version_label'.  Barry confirmed this also.